package.json

```diff
@@ -10,7 +10,8 @@
     "build": "rimraf dist && tsc",
     "test": "yarn run lint",
     "lint": "eslint \"src/**/*.{js,ts}\" --ignore-path .gitignore",
-    "migration:generate": "TS_NODE=true ts-node ./node_modules/.bin/typeorm migration:generate"
+    "migration:generate": "TS_NODE=true ts-node ./node_modules/.bin/typeorm migration:generate",
+    "migration:run": "TS_NODE=true ts-node ./node_modules/.bin/typeorm migration:run"
```

Comment:
Added this command to build migrations locally. When creating a new migration, it goes based off what your db currently looks like versus your entities. So to generate them, you want to delete your database first, then call `yarn migration:run` and then finally the normal generate command above it.